package-lock.json

```diff
@@ -1268,6 +1268,12 @@
       "resolved": "https://registry.npmjs.org/backo2/-/backo2-1.0.2.tgz",
       "integrity": "sha1-MasayLEpNjRj41s+u2n038+6eUc="
     },
```

Comment:
I guess `package-lock.json` doesn't need to be updated in this PR?

Reply:
Well, when this PR started there was no `lodash-webpack-plugin` then it was added then it was removed again. Which ended up creating new package-locks for this PR as I added and removed it back.

Reply:
I'm sorry that this PR had conflicts. I've been there myself, so I know how it feels. To prevent this frustrating go-and-forth I tend to write smaller PR that ship quickly. I see you've chosen other route, and I'd like to hear any ideas you may have to avoid/reduce this friction in the future.

Reply:
Part of that was git ignoring the dist files which are very prone to conflicts. Also here the problem was that initially I created a big PR and then split it in smaller ones which confused git and npm a bit. That should not be the case in the future.